modules/monitoring-configuring-metrics-collection-profiles.adoc

```diff
@@ -0,0 +1,35 @@
+// Module included in the following assemblies:
+//
+// * monitoring/configuring-the-monitoring-stack.adoc
+
+:_content-type: CONCEPT
+[id="configuring-metrics-collection-profiles_{context}"]
+= Configuring metrics collection profiles
+ 
+[IMPORTANT]
+====
+[subs="attributes+"]
+Using a metrics collection profile is a Technology Preview feature only. Technology Preview features are not supported with Red Hat production service level agreements (SLAs) and might not be functionally complete.
+Red Hat does not recommend using them in production. 
+These features provide early access to upcoming product features, enabling customers to test functionality and provide feedback during the development process.
+
+For more information about the support scope of Red Hat Technology Preview features, see link:https://access.redhat.com/support/offerings/techpreview[https://access.redhat.com/support/offerings/techpreview].
+====
+
+By default, Prometheus collects all metrics exposed by the default metrics targets in {product-title} monitoring components. 
```

Comment:
Adding this because even in the full profile some filtering was already being done
Also removing "monitoring" since Prometheus doesn't only collect metrics for monitoring components but all in OpenShift
```suggestion
By default, Prometheus collects all desired metrics exposed by the default metrics targets in {product-title} components. 
```